packages/client/src/runtime/core/request/createPrismaPromise.ts

```diff
@@ -26,24 +26,32 @@ export function createPrismaPromise(
 
   return {
     then(onFulfilled, onRejected, txId?: string) {
```

Comment:
I think it might be worth changing the `transactionProxy` to pass a `transaction` object here to stay consistent with `requestTransaction`.